src/open_inwoner/components/templatetags/render_tags.py

```diff
@@ -26,5 +26,5 @@ def prosemirror_content(content):
     try:
         return get_rendered_content(content)
     except (AttributeError, TypeError) as e:
-        logger.warning("Could not render content: %s", e)
+        logger.warning("Could not render content", error=str(e))
```

Comment:
I assume we can still use `exc_info=True`:
```suggestion
        logger.warning("Could not render content", exc_info=True)
```